src/components/Issues.js

```diff
@@ -13,16 +14,18 @@ import IssuesLoader from "./IssuesLoader";
 function Issues({repoName, owner}) {
   const [issues, setIssues] = useState(null);
   const [loading, setLoading] = useState(false);
+  const [issuesFilter, setIssuesFilter] = useState(false);
   const [issuesLoading, setIssuesLoading] = useState(false);
   const [cursor, setCursor] = useState(null);
   const [totalCount, setTotal] = useState(0);
   const [issuesEnabled, setIssuesEnabled] = useState(null);
   const [offset, setOffset] = useState(0);
-
+  const [totalPages, setTotalPages] = useState(0);
+  const [currentPage, setCurrentPage] = useState(0);
   useEffect(() => {
     setLoading(true);
-    api.persistedRepositoryIssuesFetch(owner, repoName).then(response => {
-      console.log(response)
+    const functionName = issuesFilter ? "persistedIssuesByLabelFetch" : "persistedIssuesFetch";
```

Comment:
Instead of `persistedIssuesByLabelFetch` use `persistedRepositoryIssuesByLabelFetch `